packaging/hipify-clang.postinst

```diff
@@ -3,8 +3,8 @@
 set -e
 
 # Soft-link to bin files
-HIPBINDIR="/opt/rocm/hip/bin"
-ROCMBINDIR="/opt/rocm/bin"
+HIPBINDIR=@CPACK_PACKAGING_INSTALL_PREFIX@/bin
+ROCMBINDIR=@CPACK_PACKAGING_INSTALL_PREFIX@/../bin
```

Comment:
Here and below: https://github.com/ROCm-Developer-Tools/HIPIFY/pull/487#discussion_r788826306

Reply:
Done.